test/unit/gateways/braintree_blue_test.rb

```diff
@@ -579,6 +579,10 @@ def test_apple_pay_card
     assert_equal "transaction_id", response.authorization
   end
 
+  def test_supports_network_tokenization
+    assert_instance_of TrueClass, @gateway.supports_network_tokenization?
```

Comment:
I want it to be explicit that it's ``true`` and not another response that passes the assertion